components/TextFields/src/MDCTextField.h

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MDCTextInput.h"
+
+/** Size of an unaltered, Material clear button. */
+extern const CGFloat MDCClearButtonImageSquareSize;
```

Comment:
nit: Based on naming, I would expect this to be a CGSize.  If all we care about for layout is the Width, consider exposing only that?
It's tricky, naming is hard.